integration_tests/mockData/prisonerByIdData.ts

```diff
@@ -14,7 +14,7 @@ const prisoners = {
         lastName: 'Jones',
         releaseDate: '2030-12-31',
         nonDtoReleaseDateType: 'HDC',
-        receptionDate: '2001-17-01',
+        receptionDate: '2001-01-17',
```

Comment:
Not related to this PR, but corrected these dates because the format was wrong! 🤦‍♂️